.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
```

Comment:
`sudo` is not required :wink: 
if you want to use python 3.7 the distribution is `xenial`

```yml
dist: xenial
language: python
python:
  - '3.7'

 services:
  - docker

 install: true

 script:
  - ./scripts/test.sh
```

Reply:
Another option with the auto release...

```yml
dist: xenial
language: python
python:
  - '3.7'

install: true

matrix:
  fast_finish: true
  include:

    - name: "Test"
      stage: test
      script: ./scripts/test.sh

    - name: "Release"
      stage: deploy
      script: ./scripts/release.sh
      # release only if tag is like #.#.#
      if: (tag =~ ^[0-9]+(\.[0-9]+){2}$)
```